pkg/writers/buffered_file_writer/bufferedfilewriter.go

```diff
@@ -285,6 +329,6 @@ func newAutoDeletingFileReader(file *os.File) *autoDeletingFileReader {
 
 // Close implements the io.Closer interface, deletes the file after closing.
 func (r *autoDeletingFileReader) Close() error {
-	defer os.Remove(r.Name()) // Delete the file after closing
+	defer os.RemoveAll(r.Name()) // Delete the file after closing
```

Comment:
Why `RemoveAll`? I thought these were individual files.

Reply:
Didn't realize I changed this. Thanks.